runExampleApp.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 PREREQUISITES=("yarn" "node")
 SCRIPT_DIR=$(
-    cd "$(dirname "$0")"
-    pwd
+  cd "$(dirname "$0")"
```

Comment:
Most changes in this file are linter/prettier. The only functional change is at line 110.